curryst.typ

```diff
@@ -80,6 +80,13 @@
   /// even if some rules have no name (unless a rule is higher than a single
   /// line).
   min-bar-height: 0.8em,
+  /// Set the orientation of the proof tree to top to bottom instead of bottom
+  /// to top.
+  ///
+  /// If set to true, the conclusion will be at the top, and the premises will
+  /// be at the bottom. The proof tree will also be aligned to the top instead
+  /// of the bottom.
+  top-to-bottom: false,
```

Comment:
Maybe this could be named `dir` and accept either `ttb` or `btt`.[^1] Using semantic values is better than using booleans imo.

[^1]: https://typst.app/docs/reference/layout/direction/

Reply:
This is definitely a better way to handle the direction! I will implement this and update the PR.